packages/eth2.0-spec-test-util/src/bulk.ts

```diff
@@ -0,0 +1,107 @@
+import {load} from "js-yaml";
+import {readFileSync, writeFile} from "fs";
+import {schema} from "./yaml/schema";
+import {describe} from "mocha";
+import {objectToCamelCase} from "./util";
+import {expect} from "chai";
+import profiler from "v8-profiler-next";
+
+export interface BaseCase {
+  description: string;
+}
+
+interface TestSpec<TestCase extends BaseCase> {
```

Comment:
Can you add a comment on what TestSpec and BaseCase are?

Eg: TestSpec corresponds to one yaml file of tests.